config/webpack.config.js

```diff
@@ -1,17 +1,15 @@
+/* eslint-disable no-var */
 'use strict';
 
-require( 'es6-promise' ).polyfill(); //needed for compatibility with older node versions
-
 var path = require('path');
-var webpack = require('webpack');
-var StatsPlugin = require('stats-webpack-plugin');
-var ExtractTextPlugin = require("extract-text-webpack-plugin");
+var GlobalConfig = require('../config/webpack.common.config');
 
 // must match config.webpack.dev_server.port
 var devServerPort = 3808;
 
 // set TARGETNODE_ENV=production on the environment to add asset fingerprints
 var production = process.env.RAILS_ENV === 'production' || process.env.NODE_ENV === 'production';
```

Comment:
also, if we have multiple config files, maybe place them all under `config/webpack/`?